lib/kovid/historians.rb

```diff
@@ -10,9 +10,9 @@ def history(country, last)
       rows = []
 
       stats = if last
-                transpose(country).last(last.to_i)
+                Kovid.format_country_history_numbers(country).last(last.to_i)
```

Comment:
Which cop that this one solve?

Reply:
I am focusing on reducing the `MethodLength` cop, but in the middle of this process I ended up moving some methods from `tablelize`, like this one. :eyes:   
I moved the methods that are intended to format the numbers to the `helpers` file, trying to put these methods that look like helpers together. This could also make `tablelize` focus on mounting table stuff like columns and rows.  
   
In this particular case while moving the `transpose` method from `tablelize` to `helpers` I renamed the method because there is already one with the same name in Ruby, so I think it could be clearer having a more particular method name related to the domain it addresses.  
  
Do you think this make sense?  
This refactoring is all in the same commit, so I can remove or modify it if you don't find it adequate.

Reply:
Cool. Good thinking.